app.vue

```diff
@@ -1,10 +1,12 @@
 <template>
```

Comment:
@nikita-badhekar @jitesh-rajput @Gaurav9938 

Kindly PLEASE NO NOT make changes anywhere in the app.js file

Please create a new page inside the pages directory if you want to check the feature that you are trying to develop!